docs/guides/securing-apps/token-exchange.adoc

```diff
@@ -6,6 +6,322 @@ title="Using token exchange"
+to test if the token looks as expected for given user and for given set of scopes and audiences.
+
+[[_standard-token-exchange-details]]
+=== Token exchange - Additional details
+
+Few more points to clarify the behaviour of token exchange.
+
+* It is not supported for public clients to send the token exchange requests. The V1 had some very limited support to public clients, when public client can exchange the token to itself with less scopes.
+This use case can be replaced by refresh token grant.
+
+* The `subject_token` sent to the token exchange endpoint must have the requester client set as an audience in the `aud` claim. Otherwise the request would be rejected. The only exception is, if client
+exchanges his own token, which was issued to it. Exchanging to itself might be useful to downscope/upscope the token or filter unneeded token audiences etc.
+
+* Consents - If requester client has `Consent required` switch enabled, the token exchange is allowed just if user already granted consent to all requested scopes
+
+* link:{adminguide_link}#_fine_grain_permissions[Fine-grained admin permissions (FGAP)] are not needed for the standard token exchange. We plan to eventually integrate with FGAP for the future, but that
+integration might be available to all grants. It will not be specific only to token exchange as it was in token exchange V1.
+
+* Integration token exchange with link:{adminguide_link}#_client_policies[Client policies] is possible. This can be useful to address the use cases like for example: Reject the token exchange request if
+client `requester-client` send the request with `scope=some-confidential-scope`. For that particular example, it can be useful to create client policy condition with combined conditions
+for `client-scope`, `grant-type` and `client-roles`.
+
+* Requesting refresh token is allowed just if client has switch `Allow refresh token in Standard Token Exchange` set to other value than `No` (which is the default value). The switch is available in the
+admin console in the tab `Advanced` of the OIDC client in the section `OpenID Connect Compatibility Modes` . The other available value of the switch is `Same session`, which means that refresh token is
+allowed just if refresh token can use the same user session like the subject token has. In case that subject token is coming from link:{adminguide_link}#_transient-session[Transient session] or from
+link:{adminguide_link}#_offline-access[Offline session], the requesting refresh token will not be allowed. Similarly it will not be allowed to request offline token (using `scope=offline_access`).
+
+.Enabling refresh token in Token Exchange
+image::token-exchange-switch-refresh.png[Enabling refresh token in Token Exchange]
+
+* Token exchange never creates new link:{adminguide_link}#managing-user-sessions[user session]. In case that `requested_token_type` is refresh token, it may eventually create new client session in the user session
```

Comment:
```suggestion
* Token exchange never creates a new link:{adminguide_link}#managing-user-sessions[user session]. In case that `requested_token_type` is a refresh token, it may eventually create a new client session in the user session
```